c2corg_ui/static/js/searchfilters.js

```diff
@@ -0,0 +1,144 @@
+goog.provide('app.searchFiltersDirective');
+goog.provide('app.SearchFiltersController');
+
+goog.require('app');
+
+
+/**
+ * Provides the 'searchFiltersDirective' directive, which is used to enrich
+ * filtering components in the documents listing pages.
+ * 
+ * @return {angular.Directive} The directive specs.
+ * @ngInject
+ */
+app.searchFiltersDirective = function() {
+  return {
+    restrict: 'A',
+    controller: 'appSearchFiltersController',
+    bindToController: true,
+    scope: true,
+    controllerAs: 'searchFiltersCtrl',
+    link: function(scope, element, attrs, ctrl) {
+      // Init sliders
+      $('.range-between').slider({min: 0, max: 9999, value: [0, 9999]});
+      
+      var elevationFilter = $('#elevation-filter');
```

Comment:
We'll do that kind of changes a bit later, when generalizing the directive for other document types etc.